vars/mavenCanaryRelease.groovy

```diff
@@ -9,10 +13,15 @@ def call(body) {
 
     def flow = new io.fabric8.Fabric8Commands()
     def utils = new io.fabric8.Utils()
-
+    def autoUpdateFMP = config.autoUpdateFabric8Plugin ?: true
     def skipTests = config.skipTests ?: false
+    FMP_STABLE_VERSION = "3.5.38"
```

Comment:
does final outside the `call` method work?

Reply:
we tried but constant didn't worked